rmgpy/molecule/draw.py

```diff
@@ -1188,21 +1213,21 @@ def __renderAtom(self, symbol, atom, x0, y0, cr, heavyFirst=True, drawLonePairs=
                     x0 = x + xadvance
 
             x = 1000000; y = 1000000; width = 0; height = 0
-            startWidth = 0; endWidth = 0
+            start_width = 0; end_width = 0
             for i, char in enumerate(symbol):
                 cr.set_font_size(self.options['fontSizeSubscript' if char.isdigit() else 'fontSizeNormal'])
                 extents = cr.text_extents(char)
                 if coordinates[i][0] + extents[0] < x: x = coordinates[i][0] + extents[0]
                 if coordinates[i][1] + extents[1] < y: y = coordinates[i][1] + extents[1]
                 width += extents[4] if i < len(symbol) - 1 else extents[2]
                 if extents[3] > height: height = extents[3]
-                if i == 0: startWidth = extents[2]
-                if i == len(symbol) - 1: endWidth = extents[2]
+                if i == 0: start_width = extents[2]
+                if i == len(symbol) - 1: end_width = extents[2]
```

Comment:
Should we change these so that they are not one liners? I could see some of these going either way